lib/http/request/body.rb

```diff
@@ -32,14 +29,10 @@ def size
       #
       # @yieldparam [String]
       def each
-        return enum_for(__method__) unless block_given?
```

Comment:
Can you please also add a comment so that in future we won't add regression:

``` ruby
# @note This method does not returns Enumerator due to known issue on JRuby:
#   https://github.com/jruby/jruby/issues/4583
```

Asking that, beacause I know at least one developer (me) who tries to sneak `to_enum __method__ unless block_given?` in every `each` method even if there's no real need in that :D